packages/angular_devkit/build_angular/src/karma/schema.json

```diff
@@ -63,6 +63,10 @@
       "type": "string",
       "description": "Defines the build environment."
     },
+    "spec": {
```

Comment:
`spec` seems quite specific to jasmine.  I think either `tests` (or maybe `files`?) would be more descriptive of the purpose of the option.
Also should this be an array of strings for additional flexibility?
Or potentially, what about an `include`/`exclude` option set for maximum configurability and similarity to other configuration files (e.g., `tsconfig`).  I would tend to favor this last option.

Thoughts @filipesilva?

Reply:
Similar pattern to TS sounds good. I'm not too keen on adding all `files/include/exclude` but maybe just `files` would be enough for now, and later `include/exclude` if needed.

Reply:
> spec seems quite specific to jasmine. I think either tests (or maybe files?) would be more descriptive of the purpose of the option.

I do not think that files is a good choice, it is not a path - it is a glob. To me `include` represents more closely to what it does and aligns to tsconfig. 

> Also should this be an array of strings for additional flexibility?

Maybe, but single value would be a good start. It is a bit more problematic as underlying lib does [not support multiple patterns](https://github.com/isaacs/node-glob/issues/217). There is a workaround with `{,,,}`

Reply:
I think `include` makes sense from the glob usage perspective.

Reply:
For list support, the glob call can be called multiple times with all the results added to a Set to de-duplicate.

Reply:
Done, renamed option to `include` and changed it's type to array. Was easier than expected, so thanks for the push!